src/ViewInstance.js

```diff
@@ -506,7 +520,7 @@ class ViewInstance extends React.Component {
       <>
         {showInventoryMenuSection && (
           <MenuSection label={intl.formatMessage({ id: 'ui-inventory.inventory.label' })} id="inventory-menu-section">
-            {canEditInstance && (
+            {canEditInstance && !isInstanceShared && (
```

Comment:
A central tenant won't have a source value started with CONSORTIUM- pref, but all instances should be shared for him and editing instance should be enabled